pkg/job/factory.go

```diff
@@ -41,7 +43,22 @@ func ConstructDockerJob( //nolint:funlen
 		GPU:    gpu,
 	}
 
-	jobInputs, err := buildJobInputs(inputVolumes, inputUrls)
+	for _, url := range inputRepos {
+		repoCID, _ := clone.RepoExistsOnIPFSGivenURL(url)
+		// if err != nil {
+		// 	fmt.Print(err)
+		// }
```

Comment:
You need to do this in a function that is expected to be long-running.

As per the past comment, implement a [job transform](https://github.com/bacalhau-project/bacalhau/tree/main/pkg/requester/jobtransform) to turn the repo into an IPFS CID if necessary. 

Also, the error should not be ignored.